### ISSUE TITLE
Update dependencies (flutter_cache_manager-3.0.0-nullsafety.1)

### ISSUE THREAD

**Issue by SmarterVision** [Baseflow/flutter_cached_network_image#578]
## 🐛 Bug Report

```
/C:/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_cache_manager-3.0.0-nullsafety.1/lib/src/storage/cache_info_repositories/cache_object_provider.dart:208:29: Warning: Operand of null-aware operation '!' has type 'String' which excludes null.
    final oldDbPath = join((await getDatabasesPath())!, '$databaseName.db');
```

### Configuration

**Version:** 2.0.3

**Platform:** 
- [x] :iphone: iOS
- [x] :robot: Android

**Comment by esase** [comment]
Yes I'm also interested in this upgrade! Thanks

**Comment by renefloor** [comment]
@esase `cached_network_image: ^3.0.0` already supports the latest versions (flutter_cache_manager: ^3.0.0). Not sure what upgrade you want, but this ticket can already be closed.

**Comment by esase** [comment]
@renefloor I guess the `pubspec.yaml` should be updated and have a :
```
flutter_cache_manager: ^3.0.0
```
instead of:

```
flutter_cache_manager: ^3.0.1
```

**Comment by renefloor** [comment]
Not really needed. I you run pub upgrade you always get version 3.0.0. Version 3.0.1 only adds support for File 6.0.0 (was File 6.1.0), so if you depend on File 6.0.0 also 3.0.1 is automatically used. Thing with `^3.0.0` is that it uses anything from 3.0.0 up to (but not including) 4.0.0. As a package maintainer you should always allow for as many versions as possible. There is no need to force package users to use 3.0.1 when 3.0.0 is fine as well. When there would be a serious bug fix in 3.0.1 that would be a reason to drop support for 3.0.0, but that's not the case here.

The original issue was about a bug in version `3.0.0-nullsafety.1`, but that is already fixed in 3.0.0.

**Comment by esase** [comment]
@renefloor ok got it! thanks and good luck
